wouso/games/teamquest/models.py

```diff
@@ -14,7 +14,10 @@ def is_head(self):
 
 
 class TeamQuest(models.Model):
-	pass
+    start_time = models.DateTimeField(default=None, blank=True, null=True)
+    end_time = models.DateTimeField(default=None, blank=True, null=True)
+    title = models.CharField(default="", max_length=100, blank=True, null=True)
```

Comment:
Similar to the above, I would be explicit in providing a name for the quest. So no default, `blank` and `null` should pe `False`.